ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1875,6 +1868,30 @@ AP_ServoRelayEvents *GCS_MAVLINK_Copter::get_servorelayevents() const
     return &copter.ServoRelayEvents;
 }
 
+#if ADVANCED_FAILSAFE == ENABLED
+AP_AdvancedFailsafe *GCS_MAVLINK_Copter::get_advanced_failsafe() const
+{
+    return &copter.g2.afs;
+}
+#endif
+
+MAV_RESULT GCS_MAVLINK_Copter::handle_flight_termination(const mavlink_command_long_t &packet) {
+    MAV_RESULT result = MAV_RESULT_FAILED;
+
+#if ADVANCED_FAILSAFE == ENABLED
+    if (super(packet) != MAV_RESULT_ACCEPTED) {
```

Comment:
Ok, so we agree there is at least one other case where AFS will say no to termination. So the actions the current code in this PR does are:

- if AFS says it will terminate, vehicle code says "*OK, then I won't disarm the motors*"
- if AFS says it won't terminate, vehicle code says "*Well, bad luck for you, I'm going to disarm the motors anyway and terminate the flight*"

For me this is an inconsistency that doesn't make sense.